Day4/que13.js

```diff
@@ -0,0 +1,23 @@
+//  Write a program to return inverse of an array
```

Comment:
What is this question about. What do we mean by inverse of an array. Why is this code returning back the same array?

Reply:
The question is about how can we inverse an array: 
Inverse of an array means that if in an array we swap the value of the array at a particular index with the index of the element, and then we obtain the same array then we can call it as the irreversible array.

eg: [3,4,2,0,1] 
in this: when we swap the array elements with the index values in another array: 

invArr = [_,_,_,_,_]
then referring the original array, we place 0 value at index 3 in invArray
invArr = [_,_,_,0,_]
Similarly when we refer the index 1 of original array we place 1 at index 4 i.e.
invArr = [_,_,_,0,1],
similarly referring the index 2,3,4 we get the inverse array: invArr = [3,4,2,0,1]